src/example.c

```diff
@@ -79,12 +80,21 @@ int main(void)
     //                               "application.log");
 
     sentry_init(options);
-    send_event();
-
-    sentry_shutdown();
 
-    // normally happens before sentry_shutdown
-    startup();
+    // Native Crash else do a Sentry Message
+    if (argc != 2) {
```

Comment:
@jan-auer can you double-check that everything in this code block looks correct? I made your suggested modifications.